middle_end/flambda/flambda_to_clambda.ml

```diff
@@ -273,8 +277,10 @@ let rec to_clambda t env (flam : Flambda.t) : Clambda.ulambda =
     to_clambda_direct_apply t func args direct_func probe dbg reg_close mode env
   | Apply { func; args; kind = Indirect; probe = None; dbg; reg_close; mode } ->
     let callee = subst_var env func in
```

Comment:
Maybe leave a CR just for the record that these layouts will be fixed in a subsequent PR